arkouda/sparrayclass.py

```diff
@@ -115,7 +117,6 @@ def __str__(self):  # This won't work out of the box for sparrays need to add th
     [array([1 1 1 ... 100 100 100]), array([17 21 29 ... 75 77 85]), array([0 0 0 ... 0 0 0])]
     """
 
-    @typechecked
     def to_pdarray(self):
```

Comment:
I think the failures for mypy are related to something in this file. Basically, I think you are creating a pdarray type that isn't supported in the pdarray creation functions, but that is just with a quick glance.

Reply:
Also, FWIW, you can run `mypy` locally to see if you are seeing the same thing

Reply:
Running `python3 -m mypy arkouda` locally gives me:

```
python3 -m mypy arkouda
Success: no issues found in 82 source files
```

Reply:
In any case, I've re-added the `@typechecked` attribute as well as a new type annotation to the `to_pdarray` method.